### DISCUSSION TITLE
Can't view files

### DISCUSSION THREAD

**Discussion by ayden600** [TumblThreeApp/TumblThree#422] [Q&A (Question & Answer)]
So I downloaded a blog on tumblthree and am trying to view the text of a downloaded post in a readable format rather than the original format. When I put it into a JSON editor, it says that it isn't properly formatted. How do I view the file? This is what it looks like in the "show files" part of the app and then when I open the file, but I want to get it in a readable format. 
![image](https://user-images.githubusercontent.com/74663762/211167071-d7242c12-df9a-418d-957b-45e4e3f57511.png)

**Comment by thomas694** [comment]
A blog's metadata can be saved in two formats, either text or json, by choosing the desired format in the blog's settings.
You should try if json format suits your needs.
